worker/baggageclaim/api/volume_server.go

```diff
@@ -541,6 +559,11 @@ func (vs *VolumeServer) StreamOut(w http.ResponseWriter, req *http.Request) {
 			return
 		}
 
+		var errExceedStreamLimit volume.ErrExceedStreamLimit
+		if errors.As(err, &errExceedStreamLimit) {
```

Comment:
Why isn't it logging it and returning just like others?

Reply:
This error check is not needed at all. I initially put the limitation check at StreamOut side, but then moved to StreamIn side. So the error check should be only done in StreamIn side.